pages/index.js

```diff
@@ -45,164 +46,170 @@ export default function Home(props) {
     }
 
     return (
-        <Layout>
-            <Hero />
-            <div className="bg-gray-100 absolute right-0 py-8 px-6 sm:px-16 sm:mt-8">
-                <div className="w-full flex justify-center absolute right-0 -top-9">
-                    <ul
-                        className="w-11/12 sm:w-1/2 flex list-none mt-3 mb-4 flex-row min-w-fit rounded-full box-border 
+      <Layout>
+        <Head>
+          <title>
+            Help UA Now | Donate to vetted, volunteer run fundraisers in
+            Ukraine.
+          </title>
+          <meta
+            key='home-description'
+            name="description"
+            content="“Help UA Now connects you to local, volunteer run fundraisers in Ukraine. Provide urgent, life-saving aid directly to those in need through one of our vetted partners. Donate to as many causes as you’d like, all in one place"
```

Comment:
There are two quotes here, causing a quote to show on the actual embed

Reply:
I pushed a fix, good catch!